build-test-db.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+export ID_MAPPER_API=http://localhost
+export ID_MAPPER_PORT=7474
+export ID_MAPPER_PASSWORD=1
```

Comment:
As per our discussion discussion it would be better to define these environment variables in a `.env` file. Which will then be picked up by `docker-compose` as well. You can then use:

```bash
set -a
source .env
set +a
```
inside of the script if you need the values. (`set -a` exports the read in variables such that they are available for the Python scripts.)